Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -91,6 +91,14 @@ struct DashboardView: View {
                     } label: {
                         Image(systemName: "plus")
                     }
+                    .accessibilityShowsLargeContentViewer {
+                        Label {
+                            Text("Add", comment: "Lable above the plus button on the DashboardView")
+                        } icon: {
+                            Image(systemName: "plus")
+                        }
```

Comment:
```suggestion
                        Label("Add", systemImage: "plus")
```

Reply:
I'm going to change this because the comment for this string can't be the same as an existing comment (that's why you see the issue in the `Localizable.xcstrings` file, I recently learned this)